www/src/pages/components/fab/android/index.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: Fab
+description: Buttons providing easy access to important actions.
+---
+
+import { graphql } from 'gatsby';
+import { ComponentHeader } from 'components/thumbprint-components';
+
+<ComponentHeader data={props.data} />
+
+### Usage
+
+FAB stands for floating action button and they perform a specific highly important task. FABS are usually high up in the page’s hierarchy and as such should be used sparingly to avoid overuse.
+
+### Components
+
+There are two types of FAB components: ThumbprintIconFab and ThumbprintTextFab
+
+ThumbprintIconFab sit in the lower right ahd side of a UI and perform important tasks that are high priority compared to other actions on the page.
+
+ThumbprintTextFab can have text and an optional icon left of the text.
+
+### Styles
+
+There is only one style of icon FABs
+
+![Icon FAB Styles](/img/android-icon-fab-styles.png 'Icon FAB styles')
+
+There are two styles for text FABs: primary and secondary
+
+![Text FAB Styles](/img/android-text-fab-styles.png 'Text FAB styles')
+
+### Implementation
+
+**ThumbprintIconFab**
+
+Configurable items
+
+1. `android:src` in XML or `ThumbprintIconFab.setImageResource()`, DrawableRes
```

Comment:
Since icon fab extends `AppCompatImageButton`, it can also support `setImageDrawable`, `setImageURI` & `setImageBitmap`. Not sure if we want to call out all of them here but at least `setImageDrawable` might be useful. @mallikapotter ?

Reply:
I added `setImageDrawable`.